package.json

```diff
@@ -189,8 +189,11 @@
     "test:integration:notice": "echo \"~~~ ⚠️ NOTICE ⚠️\n~~~ Integration tests rely on the RxPlayer build.\n~~~ Make sure you called the \\`build\\` script succesfully first.\n\"",
     "test:memory": "cross-env BROWSER_CONFIG=chrome vitest run tests/memory",
     "test:memory:chrome:watch": "cross-env BROWSER_CONFIG=chrome vitest watch tests/memory",
-    "test:unit": "vitest --config vitest.config.unit.mjs",
-    "test:unit:watch": "vitest --config vitest.config.unit.mjs --watch",
+    "test:unit": "npm run test:unit:chrome && npm run test:unit:firefox",
```

Comment:
I liked how the command was working previously:
With a small command like
`npm run test:unit array_find` A specific test can be ran very easily.

With the update, this is no longer possible, if this command is run:`npm run test:unit:chrome array_find` It will run `vitest src/ array_find` (and this will run test on all the repo..)

I think the simpler way to do this is to have a separate config for unit test (maybe extended from base one?) with: 
```
include: ["src/**/*.test.ts", "src/__tests__/**/*.ts"]
```